### DISCUSSION TITLE
Buffer mode options

### DISCUSSION THREAD

**Discussion by Andrej-Marsic** [gsuuon/llm.nvim#23] [Q&A]
Is there a way to not pollute the buffer with the input and just write there the response?
Is it possible to open the buffer in a split as a default?
Is there any chance of reutilising the properties of the original file, like filetype and syntax?

**Comment by gsuuon** [comment]
None of those are built-in but the prompt mode can be a [StreamHandlers](https://github.com/gsuuon/llm.nvim/blob/4484e1161ba20e02d5989b21823a5edb146a49dc/lua/llm/provider.lua#L31) table which you can use to do whatever you want with the response. Do you want to turn this into a suggestion?
